src/lib/components/FilterForm.svelte

```diff
@@ -80,7 +80,7 @@
       class="p-4 space-y-4"
     >
       <div class="flex flex-row flex-wrap gap-2">
-        {#each filterOptions as filterOption}
+        {#each filterOptions as filterOption (filterOption)}
```

Comment:
Do you know whether this works with arbitrary data types? From what I've seen online they mainly use an int ID for the key. I have no idea here, so lmk what you think :P

Reply:
from my testing you can stick anything in there, I think it transforms it all into a hash string, so an object would be most unique